integrations/editor_test.go

```diff
@@ -154,6 +154,24 @@ func testEditFileToNewBranch(t *testing.T, session *TestSession, user, repo, bra
 	return resp
 }
 
+func testEditFileToNewBranchAndSendPull(t *testing.T, session *TestSession, user, repo, branch, targetBranch, filePath string) *TestResponse {
+	testEditFileToNewBranch(t, session, user, repo, branch, targetBranch, filePath)
+
+	url := path.Join(user, repo, "compare", branch+"..."+targetBranch)
+
+	req := NewRequest(t, "GET", url)
+	resp := session.MakeRequest(t, req, http.StatusOK)
+	htmlDoc := NewHTMLParser(t, resp.Body)
+
+	req = NewRequestWithValues(t, "POST", url,
+		map[string]string{
+			"_csrf": htmlDoc.GetCSRF(),
```

Comment:
nit: you can use `GetCSRF(..)`, so you don't have to manually make GET request just for the csrf token.